Languages/en_US/General.php

```diff
@@ -1136,12 +1136,12 @@
 }.';
 $txt['debug_query_in_line'] = 'in {file} on line {line}, ';
 $txt['debug_query_which_took'] = '{0, plural,
-	one {which took # second}
-	other {which took # seconds}
+	one {which took {0, number, ::precision-unlimited} second}
+	other {which took {0, number, ::precision-unlimited} seconds}
 }.';
 $txt['debug_query_which_took_at'] = '{0, plural,
-	one {which took # second}
-	other {which took # seconds}
+	one {which took {0, number, ::precision-unlimited} second}
+	other {which took {0, number, ::precision-unlimited} seconds}
```

Comment:
```suggestion
	one {which took {0, number, ::precision-unlimited} second}
	other {which took {0, number, ::precision-unlimited} seconds}
```

This will give exactly three decimal places in the output every time.